### PR TITLE
Switching standard executables to live in cmd.

### PR DESCRIPTION
**Pull request by cheftako**
Moved 4 executables go programs to live in cmd.
Switched them to use Cobra to make it easy to add flags.

### CODE REVIEW

**Comment by anfernee** [comment]
those are literally examples for demo purpose, so I was just moving them from cmd to examples. 

But commands are useful. Since we are using cobra, can we merge those executables into one? maybe keep those dumb examples still for demo?

File: cmd/agentclient/main.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	}
+
+	return cmd
+}
+
+type Client struct {
+
+}
+
+func (c *Client) run() error {client := agentclient.NewAgentClient("localhost:8091")
```
Comment (anfernee) [inline cmd/agentclient/main.go]: newline.

Comment (cheftako) [reply cmd/agentclient/main.go]: Fixed

**Comment by anfernee** [comment]
lgtm.

**Pull request merged**